oxidation/crates/parsec_serialization_format/src/lib.rs

```diff
@@ -14,9 +14,17 @@ fn content(path: String) -> String {
         .parse::<PathBuf>()
         .unwrap();
     let file_path = manifest_dir_path.join(&path);
-    let mut file = File::open(file_path).unwrap();
+    let file = File::open(file_path).unwrap();
+    let buf = BufReader::new(file);
     let mut content = String::new();
-    file.read_to_string(&mut content).unwrap();
+    for line in buf.lines() {
+        let line = line.unwrap_or_else(|_| unreachable!());
```

Comment:
```suggestion
        let line = line.unwrap();
```

Why not just `unwrap` ?

Reply:
It's something that @touilleMan wants

Reply:
I've talked with @touilleMan, you're allowed in this case to use `expect` at least.

So what do you think ?

Reply:
It fails when it encounters non utf-8 character, do you need an expect ?

Reply:
Done